lua/luarocks-tag-release.lua

```diff
@@ -179,7 +201,11 @@ local function luarocks_tag_release(args)
   print(rockspec)
   print('========================================================================================')
 
-  luarocks_upload(rockspec)
+  write_file(rockspec)
+  for _, interpreter in pairs(args.luarocks_test_interpreters) do
+    luarocks_test(interpreter)
+  end
+  luarocks_upload()
```

Comment:
can it upload even if tests fail ?

Reply:
No, if `luarocks test` exits with a non-zero exit code, the pipeline will fail.